src/guide/1-meta/2-creating-project.rst

```diff
@@ -1,2 +1,4 @@
 Creating a Project
 ==================
+
+To create a new project, we recommend that you clone and use the `FLAMEGPU2 example template <https://github.com/FLAMEGPU/FLAMEGPU2-example-template>`_.
```

Comment:
..Which provides an empty model template for both the C and Py interfaces.

When configured with CMake it will automatically download and configure flamegpu (any other relevant notes from the readme, e.g. seatbelts/visualisation).

Reply:
Potentially also a note that we're working towards a pre-built python package.